packages/calling/calling/.editorconfig

```diff
@@ -0,0 +1,13 @@
+# http://editorconfig.org
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
Should we have the editorconfig out of the package?

We also already have a VS Code config file -> https://github.com/webex/webex-js-sdk/blob/master/.vscode/settings.json